src/conways_error/mod.rs

```diff
@@ -0,0 +1,5 @@
+/// Enum representing possible errors that can occur while running the game
+#[derive(Debug)]
+pub enum ConwaysError {
+    FailedToCreateCell(String),
+}
```

Comment:
I agree, what I did is add an enum `FailedToCreateCell ` to `Cell`. Although it's not being used right now because there is not much error to handle apart from user input it's gonna be useful later I believe

Reply:
Mmm what other kinds of `FailedToCreateCell` errors there could be apart from `InvalidCoordinates`? I'm not sure this enum is the right solution for this.